efficientdet/main.py

```diff
@@ -24,6 +24,16 @@
 from absl import flags
 from absl import logging
 
+try:
```

Comment:
How about putting this code under main(_), and use hparams_config to guard it?

Reply:
I have to import it before import TensorFlow to fix the GPU visible bug.Since TensorFlow doesn't support set visible devices twice.